packages/runtime/src/widgets/Flex.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+  useRegisterBindings,
+  type EnsembleWidget,
+} from "@ensembleui/react-framework";
+import { omit } from "lodash-es";
+import { WidgetRegistry } from "../registry";
+import type {
+  EnsembleWidgetStyles,
+  EnsembleWidgetProps,
+  HasItemTemplate,
+  FlexboxStyles,
+} from "../shared/types";
+import { Column } from "./Column";
+import { Row } from "./Row";
+
+interface FlexStyles extends EnsembleWidgetStyles {
+  direction?: "horizontal" | "vertical" | undefined;
+  gap?: number;
+}
+
+export type FlexProps = {
+  children?: EnsembleWidget[];
+} & HasItemTemplate &
+  EnsembleWidgetProps<FlexStyles & FlexboxStyles>;
+
+export const FlexWidget: React.FC<FlexProps> = (props) => {
+  const { values } = useRegisterBindings({
+    direction: props.styles?.direction,
+  });
```

Comment:
You need to consider what happens if you call useRegisterBindings here when the Column and Row widgets also register their own bindings. Either remove this and just evaluate the direction OR you need to destructure the important bindings, like id.